lib/use_cases/save_items_details.rb

```diff
@@ -29,9 +31,9 @@ def validation
 
   def missing_fields
     errors = []
-    @items_details.each_with_index do |row, index|
+    @items.each_with_index do |row, index|
       row.each_key do |col|
-        errors.push([:"missing_#{col}", index]) if @items_details[index][col].empty?
+        errors.push([:"missing_#{col}", index]) if @items[index][col].empty?
```

Comment:
What's col? Could this be changed to key/info?

Reply:
column? We can change it into 'field'

Reply:
col is referring to a column of the table here.
Renaming that to be 'column' instead probably makes sense.